modules/subchannel/doc/content/source/scmclosures/SCMHTCKazimiCarelli.md

```diff
@@ -0,0 +1,32 @@
+# SCMHTCKazimiCarelli
+
+!syntax description /SCMClosures/SCMHTCKazimiCarelli
+
+The HTC closure models inherit from: [SCMHTCClosureBase](SCMHTCClosureBase.md).
+
+### Kazimi-Carelli Correlation for Turbulent Nusselt Number
+
+The Kazimi-Carelli correlation [!cite](kazimi1976) is used for calculating the Nusselt number in fuel-pin bundles, considering the geometry of the bundle.
+
+\begin{equation}
+\text{Nu}_{\text{turbulent}} = 4.0 + 0.33 \times \left( \frac{p}{D} \right)^{3.8} \times \left( \frac{Pe}{100} \right)^{0.86} + 0.16 \times \left( \frac{p}{D} \right)^{5},
```

Comment:
I went to read "Clinch River breeder reactor plant. Heat transfer correlation for analysis of CRBRP assemblies"
https://www.osti.gov/biblio/7217984

and it's not called Kazimi Carelli there but FFTF.
In the Todreas and Kazimi volume 1 it's referred to as the Westinghouse correlation.

Let's add those details.
Imo the Carelli correlation might actually be the Schad-modified correlation in that first document

Reply:
>  Todreas book, chapter 10, equation 10.129

you have a newer version in this screenshot. In the one I have they list "Westinghouse". Naybe was a mistake.
Thanks!